content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Monitor application performance across hybrid cloud environment using
+  Apache SkyWalking and service mesh
+date: 2023-01-05T06:42:26.899Z
+author: Guoping Jia
+authorimage: /img/guoping.png
+disable: false
+tags:
+  - hpe-greenlake, hpe-greenlake-for-private-cloud-enterprise, Apache
+    SkyWalking, service mesh, istio, application performance monitoring
+---
+## Introduction
+
+In [the first blog](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/), we described the detailed process of deploying the complex Online Boutique application to a hybrid cloud environment, across the public EKS cluster from AWS to the private Kubernetes cluster in [HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html). This hybrid cloud model amplifies the benefits of both private and public clouds and allows for more seamless integration across technological barriers. It enables the enterprise to rely on the security of on-premises data centers while turning to the agility of managing the frontend of an application in public cloud. This model is now becoming increasingly popular as more businesses and enterprises shift toward cloud-based computing. However, this evolution presents challenges for monitoring, as applications and services are inherently more distributed in this environment. Having a good application performance monitoring (APM) tool is becoming essential within the hybrid cloud environment. It can consolidate performance metrics and troubleshooting data for assets across the hybrid cloud environment into a single application. It makes it easier to track metrics and allows enterprise to resolve problems quickly.
```

Comment:
In [a previous blog post](https://developer.hpe.com/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise/) I covered the detailed process of deploying....

Reply:
Maybe,
[In my previous blog post](https://developer.hpe.com/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh/), I covered the detailed process of deploying....

or

[In my first blog post](https://developer.hpe.com/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh/), I covered the detailed process of deploying....